auth-web/src/components/pay/LinkedShortNameTable.vue

```diff
@@ -203,13 +211,28 @@ export default defineComponent({
       }
     }
 
+    onMounted(async () => {
+      const orgSearchFilter = ConfigHelper.getFromSession(SessionStorageKeys.LinkedShortNamesFilter)
+      if (orgSearchFilter) {
+        try {
+          state.filters.filterPayload = JSON.parse(orgSearchFilter)
```

Comment:
Wouldn't it make sense to do, because it will fail if orgSearchFilter is empty?

![image](https://github.com/bcgov/sbc-auth/assets/3484109/b3da4ed8-82cf-47fa-97ce-7a5704d575cc)

```
try 
{
    state.filters.filterPayload = JSON.parse(ConfigHelper.getFromSession(SessionStorageKeys.LinkedShortNamesFilter))
}
```